### PR TITLE
[aptos] Aptos cleanup

### PR DESCRIPTION
**Pull request by m30m**

### CODE REVIEW

File: target_chains/aptos/cli/src/cli.ts [hunk 1]
```diff
@@ -1,6 +1,496 @@
 #!/usr/bin/env node
 
+import type { CommandBuilder } from "yargs";
 import yargs from "yargs";
 import { hideBin } from "yargs/helpers";
+import { spawnSync } from "child_process";
```
Comment (m30m) [inline target_chains/aptos/cli/src/cli.ts:6]: This file needs a lot more cleanup, I just moved and cleaned it up a bit.

File: target_chains/aptos/cli/README.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+--stale-price-threshold 60 \
+--update-fee 1 \
+--governance-emitter-chain-id 1 \
+--governance-emitter-address 63278d271099bfd491951b3e648f08b1c71631e4a53674ad43e8f9f98068c385 \
+--data-source-chain-ids 1 \
+--data-source-chain-ids 26 \
+--data-source-chain-ids 26 \
+--data-source-emitter-addresses f346195ac02f37d60d4db8ffa6ef74cb1be3550047543a4a9ee9acf4d78697b0 \
+--data-source-emitter-addresses a27839d641b07743c0cb5f68c51f8cd31d2c0762bec00dc6fcd25433ef1ab5b6 \
+--data-source-emitter-addresses e101faedac5851e32b9b23b5f9411a8c2bac4aae3ed4dd7b811dd1a72ea4aa71
```
Comment (ali-bahjati) [inline target_chains/aptos/cli/README.md:46]: this seems a bit brittle 🥶

File: target_chains/aptos/cli/README.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+- Create a governance proposal, proposing the aptos package to be upgraded to this specific hash
+- Approve and execute the governance proposal
+- Run the upgrade transaction and publish the new package
+
+## Generating the new contract hash:
+
+Run the following command to generate the new hash, this will assume the default deployed addresses of deployer, wormhole, and pyth, but you can override them if necessary.
+
+```bash
+ts-node cli.ts hash-contracts ../../contracts
```
Comment (ali-bahjati) [inline target_chains/aptos/cli/README.md:63]: in general i don't have ts-node installed system-wide. instead when it's on package.json i use 
`npx ts-node` and it's a bit sensitive on where the root is. I use
`npx ts-node src/cli.ts` for it it to work. (Same goes for all other ts-node usages here in this directory and elsewhere)

File: target_chains/aptos/cli/README.md (hunk 2 above)
Comment (ali-bahjati) [reply target_chains/aptos/cli/README.md:46]: Can you mention that the order matters a lot here?

**Review by ali-bahjati — APPROVED** [review]

**Pull request merged**